# HG changeset patch
# User Rik <rik@octave.org>
# Date 1524073076 25200
#      Wed Apr 18 10:37:56 2018 -0700
# Branch stable
# Node ID 1eff55e9debe8206f8bd5830e804a5a5906214c8
# Parent  cc76e2ce0a99090a0711f7a4de3bc5093117e1b4
doc: Stop cruft from being printed after "help if".

* op-kw-docs: Use TexInfo comment designator "@c" to prefix lines
which should not be shown, rather than shell character '#'.

diff --git a/libinterp/op-kw-docs b/libinterp/op-kw-docs
--- a/libinterp/op-kw-docs
+++ b/libinterp/op-kw-docs
@@ -645,24 +645,24 @@ elseif (x == 2)
   disp ("two");
 else
   disp ("not one or two");
 endif
 @end group
 @end example
 @seealso{switch}
 @end deftypefn
-## FIXME: Can't have duplicate DOCSTRING entries.  The function methods
-##        already has a docstring which overrides this keyword definition.
-#methods
-#@c libinterp/parse-tree/oct-parse.in.yy
-#-*- texinfo -*-
-#@deftypefn {} {} methods
-#Begin a methods block in a classdef definition.
-#@end deftypefn
+@c ## FIXME: Can't have duplicate DOCSTRING entries.  The function methods
+@c ##        already has a docstring which overrides this keyword definition.
+@c #methods
+@c #@c libinterp/parse-tree/oct-parse.in.yy
+@c #-*- texinfo -*-
+@c #@deftypefn {} {} methods
+@c #Begin a methods block in a classdef definition.
+@c #@end deftypefn
 otherwise
 @c libinterp/parse-tree/oct-parse.in.yy
 -*- texinfo -*-
 @deftypefn {} {} otherwise
 The default statement in a switch block which is executed when no other
 case statements match the input.
 @seealso{switch, case}
 @end deftypefn
